travis-ymls/connect-redis.travis.yml

```diff
@@ -0,0 +1,24 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : connect-redis
+# Source Repo         : https://github.com/tj/connect-redis
+# Travis Job Link     : https://travis-ci.com/github/nageshlop/connect-redis/builds/213428506
+# Created travis.yml  : Yes
+# Maintainer          : Nagesh Tarale <Nagesh.Tarale@ibm.com>
+#
+# Script License      : Apache 2.0
+#
+# ----------------------------------------------------------------------------
+dist: xenial
+arch:
+   - amd64
+   - ppc64le  
+language: node_js
+node_js:
+   - "10"
```

Comment:
why are we testing such an old node version? and only version 10?

Reply:
Will verify the changes with older and latest versions of nodejs.